packages/deployments/contracts/contracts/core/connext/facets/TokenFacet.sol

```diff
@@ -509,7 +523,7 @@ contract TokenFacet is BaseConnextFacet {
     delete s.tokenConfigs[_key].adoptedToLocalExternalPools;
     delete s.tokenConfigs[_key].approval;
     delete s.tokenConfigs[_key].cap;
-    delete s.tokenConfigs[_key].custodied;
+    // NOTE: custodied will always be 0 at this point
```

Comment:
This isn't always true:
Suppose cap & custodied have a value (`!=0`), then you call `_setLiquidityCap()` to set the cap to `0`.
The function doesn't reset the `custodied` value so it will stay at `!=0`